packages/camera/camera_android_camerax/example/android/build.gradle

```diff
@@ -16,6 +16,11 @@ buildscript {
 
 allprojects {
     repositories {
+        // See GRADLE_STRUCTURE.md for more info.
+        if (System.getenv().containsKey('ARTIFACT_HUB')) {
```

Comment:
Just a nit, instead of 'ARTIFACT_HUB' maybe be more explicit: 'ARTIFACT_HUB_REPOSITORY' or even 'ARTIFACT_HUB_GRADLE_REPOSITORY' if that isn't overly narrow.

Reply:
It also might be a good idea to make this a property (see https://docs.gradle.org/current/userguide/writing_build_scripts.html#sec:extra_properties) since it is used in two places

Reply:
Done!